explorer/explorermiddleware.go

```diff
@@ -36,18 +37,15 @@ func (exp *explorerUI) BlockHashPathOrIndexCtx(next http.Handler) http.Handler {
 				return
 			}
 		} else {
-                        //Handle blocks which are in future time
-                        maxHeight := exp.blockData.GetHeight()
-                        if height > int64(maxHeight) {
-                        maturityTime := int(float64(height - int64(maxHeight) ) * exp.ChainParams.TargetTimePerBlock.Minutes())
-                        days := maturityTime / 1440
-                        hours := (maturityTime % 1440  ) / 60
-                        minutes := maturityTime % 60
-                        message := fmt.Sprintf("This block will arrive in approximately %d days, %d hour, %d min.",days,hours,minutes)
-                        log.Errorf("Block with height %d will arrive in future.", height)
-                        exp.StatusPage(w, defaultErrorCode, message, ErrorStatusType)
-                        return
-                        }
+			//Handle future blocks
+			maxHeight := int64(exp.blockData.GetHeight())
+			if height > maxHeight {
+				expectedTime := time.Duration(height-maxHeight) * exp.ChainParams.TargetTimePerBlock
+				message := fmt.Sprintf("This block is expected to arrive in approximately in %v. ", expectedTime)
+				log.Errorf("Block with height %d will arrive in future.", height)
```

Comment:
It's still log.Error, but we don't want an ERR in the logs for this.  It's really Debugf, not an error in any way.